src/nvim/main.c

```diff
@@ -819,6 +829,14 @@ static void command_line_scan(mparm_T *parmp)
             mch_exit(0);
           } else if (STRICMP(argv[0] + argv_idx, "headless") == 0) {
             headless_mode = true;
+          } else if (STRICMP(argv[0] + argv_idx, "embed-ui") == 0) {
```

Comment:
Will this option (later) be added to `nvim --help` and the internal help?

Reply:
It will be added now.